### ISSUE TITLE
Query/Identify map tool

### ISSUE THREAD

**Issue by dan-bowerman** [fgpv-vpgf/fgpv-vpgf#270]
- [x] Format output on Data panel
- [x] Handle asynchronous loading of attributes (wait on Promise)

WMS GetFeatureInfo support will be accomplished via #283 and is considered out of scope for this issue.

Traceability Matrix ID#065

**Comment by mweech** [comment]
Partially blocked by #283

**Comment by mweech** [comment]
Block removed as WMS GetFeatureInfo support will be handled when completing #283
